frete.d.ts

```diff
@@ -7,6 +7,7 @@ declare module 'frete' {
         '40215': 'SEDEX 10 (à vista e a faturar)*';
         '40169': 'SEDEX 12 (à vista e a faturar)*';
         '40290': 'SEDEX Hoje Varejo*';
+         [key: string]: string;
```

Comment:
What are these types? Can we map them? Looks like Correios now has an [API to list services](http://ws.correios.com.br/calculador/CalcPrecoPrazo.asmx?op=ListaServicos), can we use this to get these services and map them?